### PR TITLE
First set of tests for the addon More Info card

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Tests covering this section of the card:

image

### CODE REVIEW

File: tests/test_addon_detail.py [hunk 1]
```diff
@@ -239,3 +239,65 @@ def test_extension_permissions(selenium, base_url, variables):
...
+        # opens the homepage/support page provided by the developer
+        addon.wait_for_current_url('ghostery.com')
+
+
+@pytest.mark.nondestructive
+def test_more_info_version_number(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert addon.more_info.addon_version_number.is_displayed()
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:275]: Could we check the actual version number matches?

File: tests/test_addon_detail.py [hunk 2]
```diff
@@ -239,3 +239,65 @@ def test_extension_permissions(selenium, base_url, variables):
...
+
+
+@pytest.mark.nondestructive
+def test_more_info_addon_size(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert addon.more_info.addon_size.is_displayed()
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:283]: Can we check the size matches? Or maybe some part of the text matches?

**Review by jrbenny35 — APPROVED** [review]
r+ with context

File: tests/test_addon_detail.py (hunk 2 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py:284]: @jrbenny35 I've tried to do a file size comparison in https://github.com/mozilla/addons-release-tests/pull/201/commits/21fa92bfaf25241bc6e35a12eda6929c2e062074. This seems to work well. Let me know if you have other suggestions.

File: tests/test_addon_detail.py [hunk 3]
```diff
@@ -239,3 +240,73 @@ def test_extension_permissions(selenium, base_url, variables):
...
+    more_info_size = addon.more_info.addon_size.text.split()[0].replace(' MB', '')
+    # get the file URL and read its size - conversion from bytes to Mb is required
+    file = urllib.request.urlopen(
+        'https://addons.allizom.org/firefox/downloads/file/1097275/ghostery_privacy_ad_blocker-8.5.6-an+fx.xpi')
+    size = file.length/(1024*1024)
+    # transforming the file size in two decimal format and comparing
+    # with the size number displayed in the more info card
+    assert '%.2f' % size == more_info_size
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:292]: I like it!

**Review by jrbenny35 — APPROVED** [review]

**Pull request merged**